README.md

```diff
@@ -71,6 +71,11 @@ python -m pip install "featuretools[nlp]"
 python -m pip install "featuretools[tsfresh]"
 ```
 
+**Featuretools_SQL** - Automatic EntitySet generation from relational data stored in SQL databases: 
+
+```
+python -m pip install "featuretools[sql]"
```

Comment:
Add to `install.md`:
- https://github.com/alteryx/featuretools/blob/main/docs/source/install.md